indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorIOConfig.java

```diff
@@ -205,9 +205,15 @@ public IdleConfig getIdleConfig()
     return idleConfig;
   }
 
+  @Nullable
   @JsonProperty
-  public int getStopTaskCount()
+  public Integer getStopTaskCount()
```

Comment:
nit: Would be nice to add a javadoc here telling devs to use `#getMaxAllowedStops` instead.